### ISSUE TITLE
Dynamically linked libgdx can't load libgdx64.so.

### ISSUE THREAD

**Issue by tedbarth** [libgdx/libgdx#3079]
Hi folks,

I don't know if this is a bug.

I use libgdx inside of a big maven project that compiles for pc. The project dictates me to load libgdx and its dependencies (lwjgl etc.) on runtime providing the JARs in a `/target/lib` folder and native libs in a `/target/natives` folder. My created final `visualizer.jar` includes only its own compiled classes. Everything else is added to the classpath from out `/target/lib`. I start the application via the following command on linux with various JREs:

    cd ...../target
    java -Djava.library.path=./natives -cp visualizer.jar:./lib/* org.designer.visualizer.desktop.Starter

IntelliJ has not problem starting the application without maven while extracting the natives to the target/classes directory.  Though, when starting via the above folder structure and the mentioned command it can't find the `libgdx64.so`.

In other words, `input` is null in `SharedLibraryLoader:125`:

    InputStream input = SharedLibraryLoader.class.getResourceAsStream("/" + path);

    Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Unable to read file for extraction: libgdx64.so
    	at com.badlogic.gdx.utils.SharedLibraryLoader.readFile(SharedLibraryLoader.java:126)
    	at com.badlogic.gdx.utils.SharedLibraryLoader.loadFile(SharedLibraryLoader.java:261)
    	at com.badlogic.gdx.utils.SharedLibraryLoader.load(SharedLibraryLoader.java:115)

Any advice?

**Comment by NathanSweet** [comment]
If SharedLibraryLoader.class.getResourceAsStream is null, it means the JVM
can't find the resource in the classpath. This is likely a classpath
configuration problem. Possibly the classloader that loaded
SharedLibraryLoader is different from the one that can find the .so.

**Comment by Tom-Ski** [comment]
Will reopen if self contained example is provided showing an issue with LibGDX.
